initializer/main.go

```diff
@@ -22,15 +22,34 @@ import (
 	"github.com/edgelesssys/contrast/internal/grpc/dialer"
 	"github.com/edgelesssys/contrast/internal/logger"
 	"github.com/edgelesssys/contrast/internal/meshapi"
+
+	"github.com/spf13/cobra"
 )
 
 func main() {
-	if err := run(); err != nil {
+	if err := execute(); err != nil {
 		os.Exit(1)
 	}
 }
 
-func run() (retErr error) {
+func execute() error {
+	cmd := newRootCmd()
+	ctx := context.Background()
+	return cmd.ExecuteContext(ctx)
+}
+
+func newRootCmd() *cobra.Command {
+	root := &cobra.Command{
+		Use:          "initializer",
+		Short:        "initializer",
+		RunE:         run,
+		SilenceUsage: true,
+	}
+	root.InitDefaultVersionFlag()
```

Comment:
> If c.Version is empty, it will do nothing

https://pkg.go.dev/github.com/spf13/cobra#Command.InitDefaultVersionFlag

You should be able to pass `constants.Version` though.